.github/workflows/handler.placeholder-for-testing.yml

```diff
@@ -0,0 +1,15 @@
+# Raison d'être: new handlers can't be tested before being merged to dev.
```

Comment:
good sneaky way.

Also, github allows certain secrets only be available to certain branches through "environment" secrets container, which is a good security measure for our sized projects when we have many collaborators and various computer security setup


![image](https://user-images.githubusercontent.com/186660/179720390-0cf12e33-2254-456e-a27a-47ab2e33f2da.png)